DevelopGitHub/GitHubExtension/GitHubExtension.Notes.WebApi/IdentityExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Security.Claims;
+using GitHubExtension.Notes.WebApi.ViewModels;
+using Microsoft.AspNet.Identity;
+
+namespace GitHubExtension.Notes.WebApi
+{
+    public static class IdentityExtensions
+    {
+        public static string GetUserId()
+        {
+            var userId = ClaimsPrincipal.Current.Identity.GetUserId();
+            return userId;
+        }
+
+        public static NoteModel AddUserIdToModel (this NoteModel noteModel)
+        {
+            var userId = ClaimsPrincipal.Current.Identity.GetUserId();
```

Comment:
That is not a mapper, but an extension method inside a IdentityExtensions class which I use to add UserId to a model which then will be sent to a mapper. I suppose that is a correct place to use ClaimsPricipal